training_config/bidirectional_lm.jsonnet

```diff
@@ -0,0 +1,152 @@
+local NUM_GPUS = 2;
+local NUM_THREADS = 1;
+
+local BASE_READER = {
+        "type": "simple_language_modeling",
+        "tokenizer": {
+          "type": "word",
+          "word_splitter": {
+	        # The 1 Billion Word Language Model Benchmark dataset is
+	        # pre-tokenized. (Also, if you're running against a untokenized
+	        # dataset be aware that there are serialization issues with Spacy.
+	        # These come into play in the multiprocess case.)
```

Comment:
nit: I know that jsonnet allows python-style comments, but I find them kind of jarring in "json". I don't know if anyone else cares.

Reply:
Changed to `//`. I'm impartial. Let me know if that's not what you meant!